### PR TITLE
extend the egress task

### PR DESCRIPTION
**Pull request by vadimeisenbergibm**
add checking logs, make calls more pretty, add explanations

### CODE REVIEW

File: content/docs/tasks/traffic-management/egress/index.md [hunk 1]
```diff
@@ -47,18 +47,45 @@ from within your Istio cluster. This task shows you how to access an external HT
...
+    {{< /text >}}
+
+    Note that the `destinationServiceHost` attribute is equal to `httpbin.org`. Also note HTTP-related attributes:
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/egress/index.md]: Also notice the HTTP-related ...

File: content/docs/tasks/traffic-management/egress/index.md [hunk 2]
```diff
@@ -264,8 +310,20 @@ within the cluster. Any external request bypasses the sidecar and goes straight
...
+  "headers": {
+    "Accept": "*/*",
+    "Connection": "close",
+    "Host": "httpbin.org",
+    "User-Agent": "curl/7.60.0"
+  }
+}
 {{< /text >}}
 
+Note that this time you do not see any headers related to Istio sidecar. Also note that the requests sent to external
```
Comment (frankbu) [inline content/docs/tasks/traffic-management/egress/index.md]: related to the

**Comment by frankbu** [comment]
Oh darn, I had a couple of small grammatical corrections, but forgot to not `/lgtm` so it auto merged :-(

**Comment by vadimeisenbergibm** [comment]
@frankbu I am going to submit a finalizing PR to it, with all your comments from previous PR, that were not addressed.

File: content/docs/tasks/traffic-management/egress/index.md (hunk 1 above)
Comment (vadimeisenbergibm) [reply content/docs/tasks/traffic-management/egress/index.md]: Fixed in https://github.com/istio/istio.io/pull/3222/commits/5b9ad33f13339d91904908cbab142cb4deb73197

**Pull request merged**